bank-sdk/example-app/src/androidTest/java/net/gini/android/bank/sdk/exampleapp/ui/screens/MainScreen.kt

```diff
@@ -0,0 +1,46 @@
+package net.gini.android.bank.sdk.exampleapp.ui.screens
+
+import androidx.test.espresso.Espresso.onView
+import androidx.test.espresso.action.ViewActions.click
+import androidx.test.espresso.matcher.ViewMatchers.withId
+import androidx.test.espresso.assertion.ViewAssertions.matches
+import androidx.test.espresso.matcher.ViewMatchers.isDisplayed
+import net.gini.android.bank.sdk.exampleapp.R
+
+class MainScreen {
+
+    fun assertWelcomeTitle(): MainScreen {
+        onView(withId(R.id.tv_welcomeToGini)).check(matches(isDisplayed()))
+        return this
+    }
+
+    fun assertDescriptionTitle(): MainScreen {
+        onView(withId(R.id.tv_exampleOfPhotoPayment)).check(matches(isDisplayed()))
+        return this
+    }
+
+    fun checkCheckIconDisplayed(): MainScreen {
```

Comment:
[Request] Could this method be removed if it's not used anywhere?

Reply:
Please see my comment [here](https://github.com/gini/gini-mobile-android/pull/462#discussion_r1665216027)